content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Add / Edit Store
+weight: 1
+---
+
+# Add / Edit tags 
+
+While adding or editing tags, the following fields/drop downs/parameters are displayed:
+
+1) Name 
+
+This field is for the name of the tag, the field is mandatory. When trying to save without any name input, shows the message: "The name field is required."
+
+2) Language
+
+This is a drop-down menu for the language of the tag. The default country (selected in International > Localization > Configuration) is selected by default.
+
```

Comment:
The countries listed in the drop-down are the countries enabled in International > Localization > Languages